task/sast-snyk-check-oci-ta/0.4/sast-snyk-check-oci-ta.yaml

```diff
@@ -331,7 +331,9 @@ spec:
           echo "Selecting auth"
           select-oci-auth "${IMAGE_URL}" >"${HOME}/auth.json"
           echo "Attaching to ${IMAGE_URL}"
-          retry oras attach --no-tty --registry-config "$HOME/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}@${IMAGE_DIGEST}" "${UPLOAD_FILE}:${MEDIA_TYPE}"
+          if ! retry oras attach --no-tty --registry-config "$HOME/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}@${IMAGE_DIGEST}" "${UPLOAD_FILE}:${MEDIA_TYPE}"; then
+            echo "Failed to attach to ${IMAGE_URL}"
```

Comment:
Is `exit 1` needed after this line?

Reply:
there isn't any set -e so not existing explicitly